src/Symfony/Component/HttpKernel/Tests/UriSignerTest.php

```diff
@@ -15,6 +15,21 @@
 
 class UriSignerTest extends \PHPUnit_Framework_TestCase
 {
+    /** @var string backup original value for ini setting */
+    protected $argSeparatorOutputBackup;
```

Comment:
@stloyd You mean to change it like this - https://github.com/tenerd/symfony/commit/1f40bf1a40c9c814981d842a1a3f312435a71362
?

My issue with it is that if any of the test assert fails, then ini setting is not restored to original value (so it can break other tests?)
With tearDown is this case solved.